matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/session/room/send/UserDraft.kt

```diff
@@ -24,14 +24,15 @@ package org.matrix.android.sdk.api.session.room.send
  * REPLY: draft of a reply of another message
  */
 sealed interface UserDraft {
-    data class Regular(val text: String) : UserDraft
-    data class Quote(val linkedEventId: String, val text: String) : UserDraft
-    data class Edit(val linkedEventId: String, val text: String) : UserDraft
-    data class Reply(val linkedEventId: String, val text: String) : UserDraft
+    data class Regular(val content: String) : UserDraft
+    data class Quote(val linkedEventId: String, val content: String) : UserDraft
+    data class Edit(val linkedEventId: String, val content: String) : UserDraft
+    data class Reply(val linkedEventId: String, val content: String) : UserDraft
+    data class Voice(val content: String) : UserDraft
```

Comment:
`Voice` has been added, the other diff changes are caused by `text` being renamed to `content`